examples/etcd/resources/monitor-job.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: appcontroller.k8s/v1alpha1
+kind: Definition
+metadata:
+  name: monitor-clustername-job
+meta:
+  timeout: 0
+job:
+  apiVersion: batch/v1
+  kind: Job
```

Comment:
1) Initially I wanted to use CronJob for that. By CronJobs are in alpha, not available by-default in many environments and are not supported by AC yet, so I went with regular jobs. But CronJob is still a Job so I believe Jobs are more in line here than anything else
2) ReplicaSet are less in line because using it with replica count =1 is even more hacky
3) You completely ignored my second bullet
4) As for the fact, that the job can run twice - this is possible in theory, and I believe, that for ReplicaSet it is also possible. For example, it the node failure is temporary, it may start second pod instance, but then the node gets back and now we have 2 pods running
5) Most important here: remember, that this is just an example. I don't think that endless job (replicaset etc) should be used in production